Code/Framework/AzCore/AzCore/Console/IConsole.h

```diff
@@ -94,30 +94,39 @@ namespace AZ
         //! @param commandLine the concatenated command-line string to execute
         virtual void ExecuteCommandLine(const AZ::CommandLine& commandLine) = 0;
 
+        //! Attempts to invoke a "deferred console command", which is a console command
+        //! that has failed to execute previously due to the command not being registered yet.
+        //! @return boolean true if any deferred console commands have executed, false otherwise
+        virtual bool ExecuteDeferredConsoleCommands() = 0;
+
+        //! Clear out any deffered console commands that have failed to execute from
+        //! the deferred queue
```

Comment:
Nit: this wording makes it seem like it's conditionally removing commands.  Maybe remove the `that have failed to execute` part because this queue should only contain previously failed commands.

Reply:
Yes, I'll update the comment

Reply:
also spelling on that comment, deffered -> deferred

Reply:
The typo has been fixed as well